web/gds-ui/src/components/Contact.js

```diff
@@ -47,7 +49,7 @@ const Contact = ({contact, onChange, required}) => {
         required={required}
       />
       <Form.Text className="text-muted">
-        {required ? "Required - please supply " : "Optional - if supplied, use"} full phone number with country code.
+        <Trans> {required ? i18n._(t`Required - please supply `) : i18n._(t`Optional - if supplied, use`)} full phone number with country code.</Trans>
```

Comment:
I'm very impressed at the `lingui extract` behavior, which correctly pulled out three messages for translation from the variables nested in the `<Trans>` tag -- however, I'm worried that this translation relies on English sentence structure and will produce weirdness in other languages where the order will be incorrect. This is definitely something we have to keep an eye out for when developing. 

I propose the following:

```javascript
        {required
          ? <Trans>Required - please supply full phone number with country code.</Trans>
          : <Trans>Optional - if supplied, use full phone number with country code.</Trans>
        }
```

This does duplicate the translation but allows alternative sentence structures to be used.